pkg/driver/driver_openstack.go

```diff
@@ -61,6 +62,8 @@ func (d *OpenStackDriver) Create() (string, string, error) {
 	availabilityZone := d.OpenStackMachineClass.Spec.AvailabilityZone
 	metadata := d.OpenStackMachineClass.Spec.Tags
 
+	podNetworkCidr := "100.96.0.0/11"
```

Comment:
hard-corded pod CIDR?

Reply:
exactly why is the CIDR hard-coded here?